biomes.cc

```diff
@@ -1,9 +1,11 @@
 #include "biomes.h"
 
-bool isWaterBiome(unsigned char b) {
+bool isLiquidBiome(unsigned char b) {
   return b == (unsigned char)BIOME::biOcean ||
     b == (unsigned char)BIOME::biRiver ||
+    b == (unsigned char)BIOME::biFlowingRiver ||
     b == (unsigned char)BIOME::biSwampland ||
     b == (unsigned char)BIOME::biFrozenRiver ||
-    b == (unsigned char)BIOME::biFrozenOcean;
+    b == (unsigned char)BIOME::biFrozenOcean || 
+    b == (unsigned char)BIOME::biLava;
 }
```

Comment:
I wonder if there might be a better way than just enumerating all the water-containing biomes. One approach could be to have a bit flag for "contains water" along with other bits for other biome characteristics. That would let us do things like

```
bool isLiquidBiome(unsinged int biomeDesc) { return biomeDesc & BIOME::WATER_BIT != 0; } 
```

Something like that would probably be more maintainable as we tweak the number of biomes and their characteristics.

Just something to consider - nothing that needs to block this PR.

Reply:
Will keep this in mind, thanks for the suggestion !